cpp/src/lists/explode.cu

```diff
@@ -36,6 +36,11 @@
 
 namespace cudf {
 namespace detail {
+
+// explode column gather map uses cudf::out_of_bounds_policy::NULLIFY to
+// fill nulls where there are invalid indices
+constexpr size_type InvalidIndex = -1;
```

Comment:
Wait, one last thing: people may complain about the Pascal case for the name `InvalidIndex`. Maybe you need to rename it to `invalid_index`. Waiting for other opinions....

Reply:
I'd just use an ALL_CAPS name to differentiate it.

Reply:
IMO `ALL_CAPS` should be preserved only for macro, not even for enum. If it's not enforced in the dev guide then PascalCase is perfect for this usage.

Reply:
Looking around the codebase I see examples of `INVALID_INDEX` and `invalid_index`. It seems a mixed bag. I think since it mimics a `#define` macro it should be `INVALID_INDEX`.